### DISCUSSION TITLE
v0.8.8

### DISCUSSION THREAD

**Discussion by vfsfitvnm** [vfsfitvnm/frida-il2cpp-bridge#393] [Announcements]
**Full Changelog**: https://github.com/vfsfitvnm/frida-il2cpp-bridge/compare/v0.8.7...v0.8.8

<hr /><em>This discussion was created from the release <a href='https://github.com/vfsfitvnm/frida-il2cpp-bridge/releases/tag/v0.8.8'>v0.8.8</a>.</em>

**Comment by ghost** [comment]
ok I'm glad it wasn't just my code, was inflating a list of enums, install on vm next day and whole chain blows up lmao, thanks for responding quickly
